lib/widgets/compose_box.dart

```diff
@@ -302,6 +302,7 @@ class _ContentInput extends StatelessWidget {
               style: TextStyle(color: colorScheme.onSurface),
               decoration: InputDecoration.collapsed(hintText: hintText),
               maxLines: null,
+              textCapitalization: TextCapitalization.sentences,
```

Comment:
nit in commit message:

    compose: Message compose box presents a sentence case keyboard
    
    Fixes: #487

The summary line should be in the imperative, not the indicative:
https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#commit-summary-part-2

So a direct rewording would be "compose: Present a sentence case keyboard for message compose box".

With some further tweaks, a good version would be "compose: Use sentence-case keyboard for message input".

(In particular the "compose box" is what we call the whole widget at the bottom for composing and sending a message. It includes the message input, the topic input when present, the file/photo/etc buttons, and the send button.)